tests/test_lab_init.py

```diff
@@ -20,61 +20,94 @@ class TestLabInit:
     # When using `import X`, you should use `X.Y` to patch.
     # https://docs.python.org/3/library/unittest.mock.html#where-to-patch?
     @patch("instructlab.config.init.Repo.clone_from")
-    def test_init_noninteractive(self, mock_clone_from, cli_runner: CliRunner):
-        result = cli_runner.invoke(lab.ilab, ["init", "--non-interactive"])
-        assert result.exit_code == 0
-        assert "config.yaml" in os.listdir()
-        mock_clone_from.assert_called_once()
+    def test_init_noninteractive(self, mock_clone_from):
```

Comment:
Please use the new `cli_runner` fixture. It takes care of filesystem isolation and setting `HOME` for you.

Reply:
I tried updating the tests to use it, I guess I missed a few. Thanks!